utils/rpms/daos.spec

```diff
@@ -293,6 +284,9 @@ install -m 644 utils/systemd/daos-agent.service %{?buildroot}/%{_unitdir}
 %{_bindir}/obj_ctl
 %{_bindir}/daos_gen_io_conf
 %{_bindir}/daos_run_io_conf
+# For avocado tests
+%{_prefix}/lib/.build_vars.json
+%{_prefix}/lib/.build_vars.sh
```

Comment:
Same as above, these files should not be installed into `/usr/lib/` but rather `/usr/lib/daos/`

Reply:
ok, will move the location and update .py code accordingly.

Reply:
done